index.html

```diff
@@ -15,229 +15,286 @@
-
-</section>
-
-<section class="content d-flex" id="Languages-framework-skills-forms">
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+      sed do eiusmod tempor incididunt ut labore et dolore magna 
+      aliqua. Ut enim ad minim veniam, quis nostrud exercitation 
+      ullamco laboris nisi ut aliquip ea eiusmod
+    </p> 
+    
+    <button class="download-cv-button d-flex" type="button"><span class="collaboration-button-span">Get my CV</span><img class="display-first" src="svg/ic_download.svg" alt=""><img class="display-hover" src="svg/ic_download_darktheme.svg" alt=""></button>
+  
+  </section>
+  
+  <section class="content d-flex desktop-frame" id="Languages-framework-skills-forms">
+  
+    <div class="cards first d-flex">
+      <header><h2>Languages</h2></header>
+      <ul>
+        <li>Javascript (ES5/ES6)</li>
+        <li>Ruby</li>
+        <li>HTML</li>
+        <li>CSS</li>
+      </ul>   
+        <img src="Images/ic_asterisks.jpg" alt=""/>   
+    </div>  
+  
+    <div class="cards second d-flex">
+      <header><h2>Frameworks</h2></header>
+      <ul>
+        <li>Bootstrap</li>
+        <li>Ruby on Rails</li>
+        <li>CapiBara</li>
+        <li>Selenium</li>
+      </ul>   
+      <img src="Images/ic_lamp.jpg" alt=""/>
+      
+    </div>
+  
+    <div class="cards third d-flex">
+      <header><h2>Skills</h2></header>
+      <ul>
+        <li>Codekit</li>
+        <li>CodePen</li>
+        <li>GitHub</li>
+        <li>GitLAb</li>
+        <li>Terminal</li>
+      </ul>    
+      <img src="Images/ic_skills.jpg" alt=""/>    
+    </div>
 
-  <div class="cards first d-flex">
-    <header><h2>Languages</h2></header>
-    <ul>
-      <li>Javascript (ES5/ES6)</li>
-      <li>Ruby</li>
-      <li>HTML</li>
-      <li>CSS</li>
-    </ul>   
-      <img src="Images/ic_asterisks.jpg" alt=""/>   
-  </div>  
+    <div class="cards desktop-card-fourth d-flex hide">
+      <img src="/Images/image_dots_with_teal.png" alt=""/> 
+    </div>
+  
+  </section>
+</div>
 
-  <div class="cards second d-flex">
-    <header><h2>Frameworks</h2></header>
-    <ul>
-      <li>Bootstrap</li>
-      <li>Ruby on Rails</li>
-      <li>CapiBara</li>
-      <li>Selenium</li>
-    </ul>   
-    <img src="Images/ic_lamp.jpg" alt=""/>
-    
+<div class="Desktop-Get-started">
+  <div class="contact-form-image">
+    <header class="header d-flex">
+      <img src="Images/geometry.png" alt="">            
+    </header>  
   </div>
-
-  <div class="cards third d-flex">
-    <header><h2>Skills</h2></header>
-    <ul>
-      <li>Codekit</li>
-      <li>CodePen</li>
-      <li>GitHub</li>
-      <li>GitLAb</li>
-      <li>Terminal</li>
-    </ul>    
-    <img src="Images/ic_skills.jpg" alt=""/>    
+  
+  <section class="content d-flex desktop-contact-form" id="contact-form"> 
```

Comment:
- [ ] Can you please add some more padding on top and bottom for this section? because for now the content is stuck to the top and bottom, it could be nicer to give some more padding top and bottom.
- Your design
![image](https://user-images.githubusercontent.com/55465133/130277733-f7e30488-68d6-4b18-a3f3-aa4b1b2321df.png)

- Figma design
![image](https://user-images.githubusercontent.com/55465133/130277571-e2c30a1d-52e7-4f62-a74c-5a171764ce42.png)